packages/cozy-scanner/src/DocumentQualification.spec.jsx

```diff
@@ -79,4 +94,10 @@ describe('DocumentQualification', () => {
     fireEvent.click(queryByText('Scan.items.birth_certificate'))
     expect(inputFileName.value).toBe('Manual name')
   })
+
+  it('Hide health theme', () => {
+    const { queryByText } = setup({ isFlag: true })
```

Comment:
nit: `isFlag` is not speaking for itself, but let's keep it that way since this component will be removed pretty soon.